tensorflow/python/ops/special_math_ops.py

```diff
@@ -169,7 +169,7 @@ def _enclosing_tpu_context():
 
 @tf_export('einsum', 'linalg.einsum')
 def einsum(equation, *inputs, **kwargs):
-  """A generalized contraction between tensors of arbitrary dimension.
+  """Contraction over specified indices and outer product for an arbitrary number of tensors.
```

Comment:
This looks longer than 80 characters, so our linter will likely complain

Reply:
Ah ok, if it fails I can change it to `Contraction over specified indices and outer product`.